File: src/main/java/io/bindernews/thegrackle/power/PhoenixFeatherPower.java
package io.bindernews.thegrackle.power;

import io.bindernews.thegrackle.GrackleMod;
import com.megacrit.cardcrawl.core.AbstractCreature;
import com.megacrit.cardcrawl.core.CardCrawlGame;
import com.megacrit.cardcrawl.localization.PowerStrings;
import com.megacrit.cardcrawl.powers.RepairPower;

public class PhoenixFeatherPower extends RepairPower {
    public static final String POWER_ID = GrackleMod.makeId(PhoenixFeatherPower.class);
    public static final PowerStrings STRINGS = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);

    public PhoenixFeatherPower(AbstractCreature owner, int amount) {
        super(owner, amount);
        ID = POWER_ID;
    }
}

File: src/main/kotlin/io/bindernews/thegrackle/power/PhoenixFeatherPower.kt
package io.bindernews.thegrackle.power

import com.megacrit.cardcrawl.core.AbstractCreature
import com.megacrit.cardcrawl.powers.RepairPower
import io.bindernews.thegrackle.GrackleMod

class PhoenixFeatherPower(owner: AbstractCreature, amount: Int) : RepairPower(owner, amount) {
    init {
        ID = POWER_ID
    }

    companion object {
        val POWER_ID = GrackleMod.makeId(PhoenixFeatherPower::class.java)
    }
}